us-01/Response.py

```diff
@@ -0,0 +1,73 @@
+"""
+=================================================
+Author: Drew Rinker
+Date:   09/29/21
+
+Purpose: This is the Response object class file.
+         It will define what a response is and 
+         contain variables to hold result info
+         related to each response.
+
+         *There will be more information added
+         to this class. I just don't know what 
+         that information is yet.
+=================================================
+"""
+
+class Response(object):
+    """Represents a response and related result information"""
+
+    numberOfResponses = 0
+    listOfSentimentScores = []
+
+    def __init__(self, record, recordNumber):
+        """Contructor takes in the actual record response."""
+        Response.numberOfResponses += 1
+        self.record = record
+        self.recordNumber = recordNumber
```

Comment:
I think that instead of passing in `recordNumber` to set it, it should just be set to the `numberOfResponses` after incrementing that variable. So like this:
`Response.numberOfResponses += 1`
`self.recordNumber = Response.numberOfResponses`
That way a number can't accidentally be passed in that is already used by another record and it gets done automatically.